rasa/shared/core/training_data/story_reader/markdown_story_reader.py

```diff
@@ -231,7 +232,7 @@ def parse_e2e_message(line: Text, is_used_for_training: bool = True) -> Message:
         intent = match.group(2)
         message = match.group(4)
         example = entities_parser.parse_training_example(message, intent)
-        if not is_used_for_training:
+        if not is_used_for_training and not self.use_e2e:
```

Comment:
when is `parse_e2e_message` used but `elf.use_e2e` `False`?

Reply:
when converting stories from Markdown to YAML `core.training.converters.test_story_markdown_to_yaml_converter.test_test_stories` was failing